doc/specs/vulkan/chapters/fundamentals.txt

```diff
@@ -707,13 +726,21 @@ commands, which will silently ignore these values.
 [[fundamentals-validusage-pointers]]
 ==== Valid Usage for Pointers
 
-Any parameter that is a pointer must: either be a valid pointer, or if
-_explicitly called out in the specification_, can: be `NULL`.
 A pointer is valid if it points at memory containing values of the number
 and type(s) expected by the command, and all fundamental types accessed
 through the pointer (e.g. as elements of an array or as members of a
 structure) satisfy the alignment requirements of the host processor.
 
+Valid Usage section of a command or of a relevant parameter type does state
```

Comment:
Just a few grammatical comments:

"The Valid Usage section of commands or structure types states if a command parameter or structure member requires a valid pointer."

Reply:
Yeah, that's probably better. It has a bit german sentence structure still. I might try something more direct and short like:

> If a command, or a structure type requires a valid pointer, the Valid Usage section will say so.

Reply:
+1 to @oddhack's suggestion. I think it's probably to the point enough and reads quite naturally.

Reply:
Reworded to something similar.